.travis.yml

```diff
@@ -21,9 +23,11 @@ install:
   # Theano may try to install SciPy from source if the --no-deps flags isn't
   # there.
   - pip install --no-deps Theano==0.6.0
-  - if [[ $SYMPY_MASTER == "false" ]]; then
+  - if [[ $SYMPY_VERSION == "oldest" ]]; then
       pip install sympy==0.7.4.1;
-    elif [[ $SYMPY_MASTER == "true" ]]; then
+    elif [[ $SYMPY_VERSION == "latest" ]]; then
+      pip install sympy;
+    elif [[ $SYMPY_VERSION == "master" ]]; then
```

Comment:
OH MY GOSH you can have a command take up multiple lines in the yml file???? I wish I knew this...